tests/ttnn/python_api_testing/non_working_unit_tests/grayskull/test_eltwise_polygamma.py

```diff
@@ -42,7 +42,7 @@ def run_eltwise_polygamma_tests(
 
     assert len(tt_result.shape) == len(ref_value.shape)
     assert tt_result.shape == ref_value.shape
-    assert_with_pcc(ref_value, tt_result, 0.99)
+    assert_with_pcc(ref_value, tt_result, 0.98)
```

Comment:
Are we allowed to lower PCC this way? We are usually not doing it this way for other operations.